config-connector/solutions/iam/kpt/service-account/README.md

```diff
@@ -0,0 +1,102 @@
+Service Account
+==================================================
+
+# NAME
+
+  service-account
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create a service account in your desired project, and grant it a specific role (default to iam.serviceAccountKeyAdmin) on the service account resource level.
+
+  You can also create any number of service accounts and grant them any number of roles by adding new or updating the config YAMLs and then applying them.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/member-iam member-iam
+  ```
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a service account
+  (`cnrm-system@[PROJECT_ID].iam.gserviceaccount.com`) with the following
+  roles in your desired project (it doesn't need to be the project where you
+  installed Config Connector):
+
+  - roles/resourcemanager.projectIamAdmin
+  - roles/iam.serviceAccountAdmin
+
+# USAGE
+
+  Replace `${PROJECT_ID?}` with your desired project ID value from
+  within this directory:
+
+  ```
+  kpt cfg set . project-id VALUE
+  ```
+
+  Once the project ID is set in the configs, simply apply the YAMLs:
+
+  ```
+  kubectl apply -f .
+  ```
+
+## OPTIONAL WORKFLOW
+
+  Optionally, you may want to create a different service account, grant a
+  different role to the service account, or grant multiple roles to the
+  service account, etc. Here are the instructions.
+
+**Create a Different Service Account**
+
+  Change the service account name and apply the YAMLs:
+
+  ```
+  kpt cfg set . service-account-name VALUE
+  kubectl apply -f .
+  ```
+
+**Grant a Different Role**
+
+  Change the role([predefined GCP IAM roles](https://cloud.google.com/iam/docs/understanding-roles#predefined_roles)) and apply the YAMLs. Please make sure to delete the iam policy created previously:
```

Comment:
If the target resource here is IAM Policy, then the deletion is not necessary, because the update of IAM Policy always overwrites whatever policy that is currently configured. So an update would be sufficient.

Reply:
This section is no longer necessary in the README.